subsys/net/l2/openthread/openthread.c

```diff
@@ -277,6 +277,55 @@ int openthread_send(struct net_if *iface, struct net_pkt *pkt)
 	return len;
 }
 
+static inline bool is_valid_ipv6_address(struct net_if_addr *addr)
+{
+	if (addr->is_used && addr->address.family == AF_INET6) {
```

Comment:
Why not just `return addr->is_used && addr->address.family == AF_INET6;`?

Reply:
Yep, will simplify here.  Good catch.